client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -571,7 +571,15 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
             fetchTableData();
         }
         setIsBadgeInVisible(!Boolean(Object.values(filterRules).find(item => item !== null)))
-    }, [isLoggedIn, filterRules, orderBy, currentPage, displayedCounty, userType]);
```

Comment:
are you sure these 2 need to be removed form here? why not just add a new useEffect with setCurrentPage(defaultPage); ?

Reply:
changed it like you said